playbooks/roles/configure/tasks/upload_client_certificates.yml

```diff
@@ -0,0 +1,52 @@
+---
+# DESC
+
+- name: Check if work_dir_remote has a value
+  fail:
+    msg: work_dir_remote is required
+  when: work_dir_remote is not defined or work_dir_remote is none or work_dir_remote == ''
```

Comment:
I think you have a few more variables used and you could consider using the existing task for validation eg.
```
# check variables
- import_role:
    name: common
    tasks_from: validate_variables
  vars:
    variable_list:
    - work_dir_remote
    - work_dir_local
    - inventory_hostname
```